docs/UserGuide.md

```diff
@@ -62,116 +62,68 @@ Jobby comes equipped with a [GUI]() interface that provides visual feedback to y
-the `Interview` stage.
-
-You will be able to do it simply with:
-`edit --application 5 --stage interview`
-
-![edit application](images/starter-guide/edit-application.jpg)
-
-
-This will change your `Stage` from `Resume` to `Interview`, indicating that you are onto the next stage of the application process.
-> To find more fields of the `Application` to edit, see the [edit documentation](LINK TO EDIT APPLICATION DOCUMENTATION)
-
-### Get `Reminder`
-During your internship application process, you might feel that there are many deadlines that you have to keep track of,
-in Jobby, we have a `Remind` command that will help you sort your applications by their `Deadlines`.
-
-You can do so with a simple command:
-`remind --earliest`
-to see your application deadlines from the earliest to latest.
-
-![Remind Earliest](images/starter-guide/remind-earliest.jpg)
-
-
-> To find out more ways to use `remind`, see the [remind documentation](LINK TO REMIND DOCUMENTATION).
-
-### Finding `Organization` and its related `Applications
-Let's say there is a specific organization application you are looking for, like **Woogle**, you will be able to easily find it via
-this command:
-```
-find woogle
-```
+In a job fair, you managed to meet a **Google** internship recruiter, **Josh Mao**, and he provided you with his number
+**91219121**.
 
-SCREENSHOT
+Here is how you can use [add --rec](#adding-recruiters---add---rec) command to track the Recruiter data in Jobby:
+1. type `add --rec --name Josh Mao --oid google_id --phone 91219121` into the command box
+2. press `enter`
 
-> To find out more ways to use `find`, see the [find documentation](LINK TO find DOCUMENTATION).
+![Adding Recruiter](images/ug-images/rec-added.png)
 
+You have successfully added **Josh Mao**, who is a **recruiter** from Google, whose number is **91219121**.
 
+### Adding your first Application
 
+After preparing your resume, you are ready to apply to **Google** as an intern for their **Software Engineer** role! And you know that the application deadline is on the **2-11-2023**.
 
-![List](images/starter-guide/list.jpg)
+In order to track your application, you can use [apply](#applying-to-organizations---apply) to track your application:
+1. type `apply google_id --title Software Engineer --by 22-11-2023` into the command box
+2. press `enter`
 
+![Adding Application](images/ug-images/app-added.png)
 
+You managed to add your application to **Google**! With the role you are interested in, **Software Engineer** and the application deadline which is **22-11-2023**.
 
+**Congratulations!** You have ran through the basics of Jobby. We hope that this tutorial has gave you an understanding of the 
+basic workflow of Jobby. However, there are still many features that we have yet introduced. Please refer to the [Using Jobby Section](#using-jobby) to understand
+the command structure of Jobby, or visit [Features Section](#features) to see the full capabilities of Jobby!
```

Comment:
These sentences have multiple punctuation and grammatical errors

Reply:
Ima cut the 2nd part of the first line.